src/components/group-common.ts

```diff
@@ -125,7 +125,7 @@ export const MemberCardHeaders: React.FC<MemberCardHeadersProps> = memoWithName(
 
 export interface User {
   email: string;
-  roles: string[];
+  roles: string[]; // In practice will be BillingRole or GroupRole
```

Comment:
@blakery Let me know if you have suggestions on how to handle this. I plan to work on converting the Group code next, so I can revisit how to improve the typing then.